--- Mpeg2BlockInspector_A3_APPLYABLE_CANDIDATE_v0_8.vcxproj
+++ Mpeg2BlockInspector_A3_APPLYABLE_CANDIDATE_v0_9.vcxproj
@@ -102,7 +102,6 @@
       <LinkErrorReporting>QueueForNextLogin</LinkErrorReporting>
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <ManifestEmbed>true</ManifestEmbed>
-      <ManifestInput>$(VCToolsInstallDir)Include\Manifest\segmentheap.manifest</ManifestInput>
       <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
       <ProgramDatabaseFile>$(OutDir)$(TargetName).pdb</ProgramDatabaseFile>
       <SubSystem>Console</SubSystem>
@@ -170,7 +169,6 @@
       <LinkErrorReporting>QueueForNextLogin</LinkErrorReporting>
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <ManifestEmbed>true</ManifestEmbed>
-      <ManifestInput>$(VCToolsInstallDir)Include\Manifest\segmentheap.manifest</ManifestInput>
       <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
       <ProgramDatabaseFile>$(OutDir)$(TargetName).pdb</ProgramDatabaseFile>
       <SubSystem>Console</SubSystem>
